### DISCUSSION TITLE
Should we have a regular call like OpenAPI TSC?

### DISCUSSION THREAD

**Discussion by jdesrosiers** [json-schema-org/community#23] [Governance and Policy]
I've been semi-regularly attending the OpenAPI TSC weekly calls recently and I think it's something that would be beneficial for us to do as well. I think there are several benefits.

It will help us be more organized. There are constantly things that slip through the cracks because we are all volunteers working asynchronously. If we have a time to regularly get together synchronously, we can more easily set goals, get updates on progress, identify things that still need to be done, and advocate for the issues important to us. For example, what's happening with the patch release we decided to do? Is anyone actively working on that? What still needs to be done?

It gives us a space to discuss contentious issues. Sometimes a conversation is the best way to come to an agreement or resolve a misunderstanding.

Many of us have been working together for years, but we've never met. It would be nice to get together every so often and talk about something we are all passionate about, JSON Schema. I think it can help us work together better as well. I think Henry and I would have had a better working relationship if we had ever met. I think we would have even been friends.

It could help build our community by giving people another avenue to contribute and advocate for their issues.

---

Assuming others think this is a good idea, we would need to decide the time, duration, and frequency for these calls. My preference would be for short (~30m) relatively frequent (weekly) calls. The time is the hard part because we have so many timezones involved. If we don't come up with a time that works for everyone, we can try two different times that we alternate between. That way everyone can at least make some of the calls. These calls would have no required participants and be open to anyone who has something to say or just wants to listen in.

**Comment by Relequestual** [comment]
This is much needed! Thanks Jason for starting the discussion.

30 minutes is tight, so we would need to be hyper focused, but I believe we can be!

I'm happy with a weekly meeting.

I'm going to suggest we record and publish these calls. I have created a Youtube channel.

Assuming we alternate timezone preferences, **my preference** would be at least one of the meetings within working hours, and the other after 21:00, but not on Wednesdays. GMT.
I'm also likely to have a future weekly commitment from Sept 15:00-17:00 GMT, but not sure what day yet (I'll update as soon as I know).

I have more to say but not the time today. Enough to get us going! =]

**Comment by karenetheridge** [comment]
> I'm going to suggest we record and publish these calls. I have created a Youtube channel.

I don't mind you recording the meeting for personal archiving, but if you're going to publish them publicly then I won't be participating.

**Reply by jdesrosiers** [reply]
@karenetheridge, are you ok with meetings being open to the public to join, but not recorded and shared publicly? I'd rather have your participation than have meetings recorded. However, I would like to have the ability to listen to meetings later if I have to miss for some reason or another.

**Reply by Relequestual** [reply]
Let's talk about this and see if we can work something out =]

**Reply by karenetheridge** [reply]
Yes, I have no objection to meeting recordings being shared within the organization itself, as long as they aren't published.  Publishing meeting minutes would be fine too.

**Reply by Relequestual** [reply]
I hadn't thought this would be an issue.
There's no expectation of cameras being on, nor to use audio.
It's perfectly fine to only use chat.

Even if we didn't make recordings public, as it would be an open meeting, so anyone could join, record, and publish, if they were so inclined.

**Reply by Relequestual** [reply]
I'm still not sure I understand the objection. I'm hoping we can convince you to join in some way regardless.
I'm going to suggest we move forward with the plan to record and upload to youtube an see how it goes.

@jdesrosiers want to suggest some times as a new comment? Then we can promote and ask the wider community!

**Reply by jdesrosiers** [reply]
I still think @karenetheridge's participation is far more valuable than publishing recordings publicly. I hope we can work something out.

**Reply by Relequestual** [reply]
There's no requirement to use video or audio on the calls, and being on a call or not does not prohibit participation in the project overall. The calls will allow us to move some things forward faster.
I really hope we can win you over on this @karenetheridge =]

**Reply by m-mohr** [reply]
Why publish them to YouTube though? Can't they just reside in some kind of cloud storage such as Dropbox and being shared on request?

**Reply by Relequestual** [reply]
Under advice from those that are very much more in open source at large than I, not making the rcordings pubilcally accessible will make it look like a closed meeting, even if it isn't.

Making the recordings avilable on youtube will show clearly that there are public meetings happening.

We could just make the recording from Zoom available in the meeting issue like OpenAPI do, but if JSON Schema stopped being supported by Postman or I left for some reason, access to the recordings could go away.
I think having them on something like dropbox might end up taking a lot of space.

Additionally, showing that we have active meetings that people can easily watch demonstrates our open nature, making the project more approachable.

**Reply by m-mohr** [reply]
On the other hand, you may discourage people from participating. I'd be interested in participating from time to time, but I'd not do it if they show up on YouTube. I'm also participating in recorded calls at the OGC, but those are at least limited access to OGC members. Some people may just not want to be publicly shared on platforms such as YouTube.

**Reply by Relequestual** [reply]
Could you help me understand why you would see this as any different to participating in public GitHub Discussions like this one? There is no obligation or expectation to use video.

**Reply by m-mohr** [reply]
Some reasons, which don't necessarily all apply to me:
1. Social phobia increases if things get published on platforms such as YouTube, which likely even increases for non-native speakers (writing is different than speaking).
2. Concerns about YouTube itself (e.g. their policies or people ranting in comments).
3. Some people just don't want that their participation (and rambling) to be completely public to everyone.
4. Concerns about employers spotting it for various reasons

**Reply by fmvilas** [reply]
From my experience with [AsyncAPI public meetings](https://www.youtube.com/watch?v=pO9bcs7l3fQ&list=PLbi1gRlP7pijUwZJErzyYf_Rc-PWu4lXS):

> Social phobia increases if things get published on platforms such as YouTube, which likely even increases for non-native speakers (writing is different than speaking).

While I agree, I found out that people who are shy to speak because they're not good at English, usually don't even join the meetings that are not recorded.

> Concerns about YouTube itself (e.g. their policies or people ranting in comments).

What policies do you think people would be worried about? Regarding people ranting in comments, it's easy, disable them. However, I must say nobody ever ranted on our channel.

> Some people just don't want that their participation (and rambling) to be completely public to everyone.

Well, that's a problem. This is an open-source project. Everything should be happening transparently and in public. If someone doesn't like their participation to be available to everyone they're simply in the wrong place.

> Concerns about employers spotting it for various reasons

What could an employer be concerned about exactly? I'm curious.

**Reply by Relequestual** [reply]
We want to be inclusive. We invite people to join and use chat only, while also being able to listen in on discussions, if they so wish. There's no reason anyone has to use either video or audio during the calls to participate. I hope deliberatly making space for people who make those choices for whatever reason is enough to eliviate some level of social phobia. We are however an open organisation as much as possible.

Comments will be disabled. As meetings will have an issue, nessecary comments will go there.
As to their policies, one might equally be concerned about GitHub policies, and many have moved elsewhere as a result. That is a valid choice. If there's a preferable solution to YouTube, I'm willing to listen. I have similar concerns about Dropbox personally.

I don't see how 3 makes a difference if anyone can request to view and download the videos anyway?

I don't know how to respond to no. 4.

---
I want to be really clear.

Attending these calls is not required to contribute nor to have significant impact on the project. They would be a means for maintainers to check in, review issues, feedback on PRs, and move things forward, at a faster rate than is otherwise possible.

Not attending the calls for any reason does not exclude anyone from our community.

Those from the community who choose for whatever reason to not join the calls are not going to be given any less consideration.

Our Slack server is vibrant, and we have many people from the community ask for and receive help, and contribute to the specification and project.

There are several individuals that do not attend the OpenAPI calls and yet have had significant impact on the project.

We are not going to suddenly ignore Issues or Pull Requests. If anything, the calls will help us respond and resolve them faster than before.

There is an optics aspect to this. I feel very strongly that JSON Schema as a project needs to be **seen** more if it's going to be effective. I outlined the reasoning in my recent talk on [the state of JSON Schema](https://www.youtube.com/watch?v=im_uu_9p7Jo).

**Reply by m-mohr** [reply]
While I appreciate your responses, I won't go into further details here. I actually did not expect any responses to any of my points mentioned above. My primary aim was to second Karens point and say that not everyone will be happy with YouTube publishing and thus some attendees such as Karen and me might get excluded. What I'm surprised about is why the alternatives Karen mentioned were not considered/discussed.

Out of curiosity, could you link me to any other open-source project that publishes their calls on YouTube or a similar platform? I've never seen any OS project do that before.

I never said or meant to say that those calls would take anything away from participation etc. Are you sure that hour-long calls on YouTube are a good way to make JSON Schema much more visible? I don't think so, but that's just my 2ct.

> I found out that people who are shy to speak because they're not good at English, usually don't even join the meetings that are not recorded.

From personal experience in the past: Sometimes it's just about having a low barrier for people to join and then they feel comfortable and get more and more active. Having public YouTube recordings in place could be quite a barrier for some.

**Reply by fmvilas** [reply]
From the top of my head, you have:

* [Kubernetes](https://www.youtube.com/c/KubernetesCommunity/playlists)
* [Kyma](https://www.youtube.com/channel/UC8Q8bBtYe9gQN-dQ-_L8JvQ/playlists)

I know some other Linux Foundation and CNCF projects are doing it. In any case, I think it's cool to have them and proved useful for AsyncaPI too. It's not mandatory to join to become a contributor, so those who are not comfortable can simply not join and they're still welcome. The low barrier entry is the Github issues and discussions.

**Reply by m-mohr** [reply]
Interesting, I've only been involved in OS projects with written meeting notes or recoded calls that are only available to "members".

> It's not mandatory to join to become a contributor, so those who are not comfortable can simply not join and they're still welcome.

I fully disagree, but we won't reach a consensus here it seems (and we don't need to).

> The low barrier entry is the Github issues and discussions.

Sorry to be not 100% clear here, I meant low barrier for calls. Calls are much more personal and are a different level to become a member of a community than issues and discussions.

**Comment by jdesrosiers** [comment]
It looks like there's a time we might be able to squeeze everyone in at the same time.

08:00 NZST
13:00 PST
16:00 EST
21:00 GMT

I imagine this is after @Relequestual's kids go to bed, but not too late while hopefully not too early for @gregsdennis and in the middle of the day for those of us in North America. If we have anyone who wants to join at some point from Eastern Europe, it will be pretty late, but hopefully not too unreasonable. It would be really late for someone from India to join, but we don't have anyone from that region right now.

Let's start negotiations there. If we can make that time work, it would be best because everyone can be there.

**Reply by gregsdennis** [reply]
What day are you looking at? That will determine my ability to join as I might be headed into work.

**Reply by Relequestual** [reply]
*looks up time zones*
For me evening wise, it would need to be Monday, Tuesday, or Friday. Would be the following day for you, so Tuesday, Wednesday, or Saturday. I'm guessing Sat wouldn't be ideal for you? 😅

If we could do two calls a month (or every other week), where one is that time and one is in working hours for europe, that would be preferable.

**Reply by gregsdennis** [reply]
Saturday is actually probably better as I won't be on a bus or walking to the office.

I can't take time from work to do this.

**Reply by jdesrosiers** [reply]
Friday works for me.

**Reply by Relequestual** [reply]
Given @gregsdennis has recently declared a step back, I'm guessing regular meeting calls might be off the table for him.
As such, I'm going to suggest a few other times, which would make it much more viable for me, and be within working hours for EU and US.

**Reply by Julian** [reply]
4 EST working days would normally be a decent time for me too.

**Reply by fmvilas** [reply]
In case it helps, at AsyncAPI we're alternating the meetings between APAC and AMER-friendly times each week. This is working well because we never take decisions during the meeting but we rather use this time for the community to chime in. or explain things in a video format to support the written form. This way, we don't need everybody to be in the meeting at the same time.

**Comment by Relequestual** [comment]
OK, so I think what I want to propose is a twice monthly rotating timezone schedule.

First Friday of the month (US/NZ friendly):
08:00 NZST
13:00 PST
16:00 EST
21:00 GMT
(https://everytimezone.com/s/1b563ec7)

Third Friday of the month (UK/EU friendly):
02:00 NZST (I know)
07:00 PST
10:00 EST
15:00 GMT
(https://everytimezone.com/s/d2d8b24e)

Please thumbs up or thumbs down and/or leave a reply comment.

**Reply by Relequestual** [reply]
This was formalised in UTC https://twitter.com/jsonschema/status/1427214613439975424?s=20

Spiral calendar padClock face three oclock Twice monthly
First Friday - 20:00 UTC
Third Friday - 14:00 UTC
